frontend/benefit/applicant/src/context/AppContext.tsx

```diff
@@ -4,15 +4,11 @@ import React from 'react';
 import { SubmittedApplication } from '../types/application';
 
 export type AppContextType = {
```

Comment:
Usually react-query caches the latest submitted application, so I think only the id is needed to save to the context and then use `useApplicationQuery(id)` to get the cached value. 
Also Context has the problem that it vanishes if page is reloaded (also reload happens now when changing language). query url parameter or localstorage could be more persistent.

Reply:
This is anyway more like a comment, not request for a change :)